apps/frontend/src/pages/register.tsx

```diff
@@ -166,6 +169,22 @@ const Register: React.FC = () => {
   };
 
   const handleRegisterWithPassword = async (password: string) => {
+
+    // Check password strength
+    const passwordCheck = zxcvbn(password);
+    if (passwordCheck && passwordCheck.score < 3) {
+      let errorMsg: string = "Weak password, submit new one.";
+      if (passwordCheck.feedback?.warning) {
+        errorMsg += ` ${passwordCheck.feedback.warning}.`;
+      }
+      if (passwordCheck.feedback.suggestions.length > 0) {
```

Comment:
i feel like feedback suggestions for a password, esp displayed as a toast, doesn't send the right ux message. this is more of a master password than an auth password, so we should just have the user retry